app/src/lib/git/diff.ts

```diff
@@ -10,6 +10,72 @@ import { DiffType, IRawDiff, IDiff, IImageDiff, Image } from '../../models/diff'
 
 import { DiffParser } from '../diff-parser'
 
+function wrapAndParseDiff(args: string[], path: string, name: string, callback: (diff: IRawDiff) => Promise<IDiff>, successExitCodes?: Set<number>): Promise<IDiff> {
+
+  return new Promise<IDiff>((resolve, reject) => {
+    const commandName = `${name}: git ${args.join(' ')}`
+    log.debug(`Executing ${commandName}`)
+
+    const startTime = (performance && performance.now) ? performance.now() : null
+
+    const process = GitProcess.spawn(args, path)
```

Comment:
It'd be nice to standardize on `spawn` so that we only have one codepath, but that can wait. Mind opening an tech debt issue to track it tho?